### DISCUSSION TITLE
Play in Japanese

### DISCUSSION THREAD

**Discussion by john-savepoint** [tsunamods-codes/7th-Heaven#134] [Ideas]
Hello, I play FF7 modded every year for years now. Such awesome work!!! The last couple of years have seen insane improvements. I look forward to playing this with my kids.

I am living in Japan permanently now and would love to play this in Japanese to learn. It seems there is no Japanese language version. Is there?

If not, is this an opportunity for me to join this community as a modder to add a Japanese version mod? There are full Japanese version game scripts available.

**Comment by julianxhokaxhiu** [comment]
Hello @john-savepoint, thanks for your kind words :) The current main blocker that is not allowing us to play FF7 in Japanese ( or say any Unicode language ) is this one: https://github.com/julianxhokaxhiu/FFNx/issues/39 If you live in Japan and you know whom to approach there to get the issue fixed you're more than welcome. Feel free to join that issue thread to discuss this one further. Cheers!

**Reply by john-savepoint** [reply]
Awesome thank you for your response. I have some friends that might be able to help but this is a great start already, thank you.
